source/account_structure/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Account structure 
+weight: 55
+---
+
+# Account structure 
+
+If you have multiple services integrated, you can treat them separately or as
+a combined single service. You can use the GOV.UK Pay platform to set this this within
+GOV.UK Pay and within your Payment Service Provider (PSP). 
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/account_structure.png)
+
+You may have 2 services, for example for parking permits and parking fines.
+The diagram takes this example and illustrates the different possible
+structures you can have:
+
+### Type 1 (GOV.UK Pay separate, PSP separate)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+both GOV.UK Pay and the PSP.
```

Comment:
Might want to add that they can be paid into separate bank accounts since they also have their own PSP accounts.

Reply:
Done